examples/graphql/schema.js

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+/* eslint-disable no-unused-vars */
```

Comment:
curious, which variable is complaining about here that made you add this disable?